Library/Homebrew/linkage_checker.rb

```diff
@@ -51,13 +53,15 @@ def display_reverse_output
   def display_test_output(puts_output: true)
     display_items "Missing libraries", @broken_dylibs, puts_output: puts_output
     display_items "Broken dependencies", @broken_deps, puts_output: puts_output
+    display_items "Unwanted system libraries", @unwanted_system_dylibs, puts_output: puts_output
```

Comment:
Can this get not shown on macOS if it's empty? Up to you what happens on Linux in the empty case.